### DISCUSSION TITLE
check how many listeners is registered?

### DISCUSSION THREAD

**Discussion by artstylee** [socketio/socket.io#4220] [Q&A]
hi
is there any way to check how many listeners is registered both on backend and frontend?
just wanna debug the whole code and check if i made a mistake and some listeners got duplicated over and over again

**Comment by darrachequesne** [comment]
Hi! You can check the number of listeners with `socket.listeners("my-event").length`

Reference: https://nodejs.org/api/events.html#emitterlistenerseventname
